### PR TITLE
Add eliseofferman coordinates

### PR DESCRIPTION
**Pull request by eliseofferman**
I want to add my pin to the map

### CODE REVIEW

File: _pins/eliseofferman.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: eliseofferman
+latitude: 59.3333
+longitude: 18.05
```
Comment (Johannahok) [inline _pins/eliseofferman.json]: @eliseofferman good job!

Comment (mattiasa1974) [inline _pins/eliseofferman.json]: @eliseofferman Looks great!

Comment (eliseofferman) [reply _pins/eliseofferman.json]: Tack!!!

Comment (eliseofferman) [reply _pins/eliseofferman.json]: 👍

**Pull request merged**